.github/workflows/format.yml

```
@@ -9,18 +9,20 @@ on:
 jobs:
   formatting-check:
 
-    name: Formatting Check
+    name: Lint
     runs-on: ubuntu-latest
 
     steps:
-    - uses: actions/checkout@v2
+    - uses: actions/checkout@v3
+    - uses: actions/setup-python@v4
+      with:
+        python-version: 3.9
     - name: Install dependencies
       run: |
-        pip install toml yapf
-    - name: run YAPF to test if python code is correctly formatted
+        python -m pip install --upgrade pip
+        python -m pip install poetry
+        poetry install
+    - name: Run yapf and isort
       run: |
-        yapf --diff --recursive --verbose --exclude=**/docs/* $GITHUB_WORKSPACE
-    - uses: actions/setup-python@v2
-      with:
-        python-version: 3.9
-    - uses: jamescurtin/isort-action@master
+        poetry run tox -e yapf
+        poetry run tox -e isort
```

[nits] If both yapf and isort is failed, the following method display both errors.

```suggestion
        poetry run tox -e yapf,isort
```

---

@kitagry Thanks for your suggestion! Applied.